static/src/js/util/createSpatialDisplay.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Returns a string displaying the spatial information.
+ * @param {Object} spatial Object that holds the different spatial areas.
+ * @returns {String} Returns a string formatting the spatial areas into human readable values.
+ */
+export const createSpatialDisplay = (spatial) => {
+  const {
+    boundingBox,
+    circle,
+    line,
+    point,
+    polygon
+  } = spatial
+
+  const selectedShape = boundingBox || circle || line || point || polygon
+
+  if (selectedShape) {
+    const splitStr = selectedShape[0].split(',')
+
+    if (boundingBox) { return `SW: (${splitStr[1]}, ${splitStr[0]}) NE: (${splitStr[3]}, ${splitStr[2]})` }
```

Comment:
SpatialDisplay.js has methods that are doing the same thing you are doing here (or at least they are getting the points into the right order), you should pull those methods out into their own utils and reuse them here.

transformSingleCoordinate
transformBoundingBoxCoordinates
transformCircleCoordinates

Reply:
@macrouch This does not make the output look very good... 
instead of this
SW: (-9.56677, 105.57422) NE: (-8.23114, 107.40234)
we get
(-9.56677,105.57422,-8.23114,107.40234)

etc.

Reply:
Right, those methods aren't doing everything you need, but we don't need to duplicate what those methods are already doing, which is ordering the points in the string correctly.